src/views/Main/Main.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import CountryCard from '../../components/CountryCard/CountryCard';
+import Select from '../../components/Select/Select';
+import { fetchCountries } from '../../services/countries';
+import './Main.css';
+
+export default function Main() {
+  const [countries, setCountries] = useState([]);
+  //for selected continent
+  const [continent, setContinent] = useState('All');
+  //to create array of continents
+  const [continents, setContinents] = useState([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const resp = await fetchCountries();
+        setCountries(resp);
+        //put the continent values from countries into array
+        const contList = [...new Set(
+          resp.map((country) => (country.continent)
+          ))];
+        //filter out null
+        setContinents(contList.filter(Boolean));
```

Comment:
great work on using .map + .filter, these are helpful methods as you build more complex apps.